ports/libassert/portfile.cmake

```
@@ -1,37 +1,30 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO jeremy-rifkin/libassert
-    REF "v${VERSION}"
-    SHA512 beba94e033f7e43c84123736a32725a333c915392d5dc57c26a63f832a507564d79f290a151cb136de8bded3d8d343dad3c4bf2efec9977d878df3c9a8677554
+    REF "v${VERSION}-beta"
+    SHA512 cf6e84caf7fbda5a2bc5a89a5ce4fb388283d5f392b60b3247dee89b0ab5a46adbc84d8562febc9d17e97863b7d35e0142e05ba0abeb02d31199c800c5ba72ba
     HEAD_REF main
-    PATCHES
-        runtime_destination.patch
-        magic-enum.patch
-        include-dir.patch
-        cpptrace-0.4.1.patch
 )
 
-string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" BUILD_STATIC)
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dyanmic" BUILD_SHARED)
```

```suggestion
string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" BUILD_SHARED)
```
Typo. Sorry :facepalm:

---

Oops thanks